pkg/arvo/gen/azimuth-sources.hoon

```diff
@@ -0,0 +1,4 @@
+:-  %say
+|=  [[now=@da *] *]
+:-  %noun
+.^(state-eth-node:jael j//=sources/(scot %da now))
```

Comment:
wh- why is this valid path syntax?

And why not just the more common `%j /=sources` (or even `/j/=sources`) here?

Reply:
You can't do `/j/=sources` because the `=`'s don't get counted right (ie it becomes %home instead of the ship name).

This syntax was considered best practice once upon a time because the vane is "a part of the path" in the sense that it's required to make it a referentially transparent name.  I still prefer that, but I guess that's changed.